tests/container_disk_test.go

```diff
@@ -54,10 +53,8 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 		util.PanicOnError(err)
 	})
 
-	verifyContainerDiskVMI := func(vmi *v1.VirtualMachineInstance, obj runtime.Object) {
-		_, ok := obj.(*v1.VirtualMachineInstance)
-		Expect(ok).To(BeTrue(), "Object is not of type *v1.VirtualMachineInstance")
-		tests.WaitForSuccessfulVMIStart(obj)
+	verifyContainerDiskVMI := func(vmi *v1.VirtualMachineInstance) {
```

Comment:
This change does not seem to be related to the commit.

Reply:
It's a small fixup that for simplicity I've included in the commit.

Reply:
I think it will be cleaner if you move it to a separate commit.